script/playbackstrategy/msestrategy.js

```diff
@@ -73,12 +84,30 @@ define('bigscreenplayer/playbackstrategy/msestrategy',
       function onQualityChangeRendered (event) {
         if (event.mediaType === 'video') {
           DebugTool.info('ABR Change Rendered from: ' + event.oldQuality + ' to: ' + event.newQuality);
+
+          if (!bitrateInfoList) {
+            bitrateInfoList = mediaPlayer.getBitrateInfoListFor('video');
+          }
+          if (bitrateInfoList && event.newQuality) {
+            playerMetadata.playbackBitrate = bitrateInfoList[event.newQuality].bitrate / 1000;
+          }
+          Plugins.interface.onPlayerInfoUpdated(playerMetadata);
         }
       }
 
       function onMetricAdded (event) {
-        if (event.mediaType === 'video' && event.metric === 'DroppedFrames') {
-          DebugTool.keyValue({key: 'Dropped Frames', value: event.value.droppedFrames});
+        if (event.mediaType === 'video') {
+          if (event.metric === 'DroppedFrames') {
+            DebugTool.keyValue({key: 'Dropped Frames', value: event.value.droppedFrames});
+          } else if (event.metric === 'BufferLevel') {
```

Comment:
Do we not want to pull this outside of the `if(event.mediaType === 'video')`? Do we get mediaType audio? And if so, we would probably want to call the plugin.

Reply:
Should now report for audio too, since the code is no longer behind some logic i'm not updating the tests as it should function the same for both audio and video.